src/server/failover.c

```diff
@@ -119,6 +119,7 @@ extern char	     server_host[];
 
 extern struct connection *svr_conn;
 extern struct batch_request *saved_takeover_req;
+struct sched_socks scks = {0};
```

Comment:
No need to initialise the structure early.

Reply:
done